pages/docs/workflows.md

```diff
@@ -5,14 +5,10 @@ next-section = "docs/cdn"
 prev-section = "index"
 +++
 
-# Workflow Guides
+# Generating Import Maps
```

Comment:
This title is the title of the entire page, are you suggesting renaming this entire section to `https://jspm.org/generating-import-maps` instead of `https://jspm.org/workflow-guides`? Alternatively, we could make this a sub-section.